frontend/src/metabase/browse/components/BrowseMetrics.tsx

```diff
@@ -0,0 +1,84 @@
+import { t } from "ttag";
+
+import NoResults from "assets/img/metrics_bot.svg";
+import { useFetchMetrics } from "metabase/common/hooks/use-fetch-metrics";
+import EmptyState from "metabase/components/EmptyState";
+import { DelayedLoadingAndErrorWrapper } from "metabase/components/LoadingAndErrorWrapper/DelayedLoadingAndErrorWrapper";
+import { Box, Flex, Group, Icon, Stack, Text, Title } from "metabase/ui";
+
+import type { MetricResult } from "../types";
+
+import {
+  BrowseContainer,
+  BrowseHeader,
+  BrowseMain,
+  BrowseSection,
+} from "./BrowseContainer.styled";
+import { MetricsTable } from "./MetricsTable";
+
+export function BrowseMetrics() {
+  const metricsResult = useFetchMetrics();
+  const metrics = metricsResult.data?.data as MetricResult[] | undefined;
+
+  const isEmpty = !metricsResult.isLoading && !metrics?.length;
+
+  return (
+    <BrowseContainer>
+      <BrowseHeader role="heading" data-testid="browse-metrics-header">
+        <BrowseSection>
+          <Flex
+            w="100%"
+            h="2.25rem"
+            direction="row"
+            justify="space-between"
+            align="center"
+          >
+            <Title order={1} color="text-dark">
+              <Group spacing="sm">
+                <Icon size={24} color={"var(--mb-color-brand)"} name="metric" />
+                {t`Metrics`}
+              </Group>
+            </Title>
+          </Flex>
+        </BrowseSection>
+      </BrowseHeader>
+      <BrowseMain>
+        <BrowseSection>
+          <Stack mb="lg" spacing="md" w="100%">
+            {isEmpty ? (
+              <MetricsEmptyState />
+            ) : (
+              <DelayedLoadingAndErrorWrapper
+                error={metricsResult.error}
+                loading={metricsResult.isLoading}
+                style={{ flex: 1 }}
+                delay={0}
+                loader={<MetricsTable skeleton />}
+              >
+                <MetricsTable metrics={metrics} />
+              </DelayedLoadingAndErrorWrapper>
+            )}
+          </Stack>
+        </BrowseSection>
+      </BrowseMain>
+    </BrowseContainer>
+  );
+}
+
+function MetricsEmptyState() {
```

Comment:
This is a really nice empty state. It advertises usage of Metrics.
It's a shame that almost no one will ever see it because we don't link to this page unless there are metrics already.

@mngr should we reconsider and show the link to this page in the sidebar even if there are no metrics?

Reply:
> This is a really nice empty state. It advertises usage of Metrics. It's a shame that almost no one will ever see it because we don't link to this page unless there are metrics already.
> 
> @mngr should we reconsider and show the link to this page in the sidebar even if there are no metrics?

Yes, I answered this already a while ago, yes, we should show this page even if there are no metrics in the instance